### ISSUE TITLE
Me: Enable choosing profile picture (Gravatar) from Site Media Library and not (just) Device Library

### ISSUE THREAD

**Issue by erricgunawan** [wordpress-mobile/WordPress-iOS#14105]
### Expected behavior

After tapping on the Profile Picture to add/change it, there should be an option to pick an image from the Site's Media Library.

### Actual behavior

Currently, it will display all pictures from the Device Library instead (besides Camera Roll).

Looks like defaulting to the Selfies album is the intended behavior per #6153.

Also, there's no option to pick from Media Library in the web/desktop app version.

### Steps to reproduce the behavior

1. Select the ‘Me’ tab.
2. Tap the user profile picture.
3. The device's library (Albums) will be displayed. No option to select an image from Media Library.

**Comment by rachelmcr** [comment]
Thanks for the report! This is expected behavior, since your profile picture (Gravatar) isn't linked to your site or site's media library directly.

That said, given that most users have a single site and we're moving the Me tab to the My Site section of the app (thus strengthening the perceived link between the site and the Me section), this is an interesting enhancement to consider. I'm not sure what backend implications there would be for how the Gravatar is saved.

**Comment by thehenrybyrd** [comment]
I ran into this as a disappointing experience during a usability exploration: pdcxQM-1ff-p2
After creating an issue (https://github.com/wordpress-mobile/WordPress-iOS/issues/19582) I found this one, so closing my issue as a duplicate.

The fact that we're using Gravatar (a separate service) for these profile icons does complicate the situation, but I suspect most users aren't aware of Gravatar. Perhaps we could simply upload items from the Media Library to Gravatar if the user chose such an image.
